tests/5.0/simd/test_simd_nontemporal.c

```diff
@@ -0,0 +1,50 @@
+//===--- test_simd_nontemporal.c ------------------------------------------------------------------------===//
```

Comment:
This header runs a bit long, I think we try to keep the line length always no greater than 82 characters (for narrow screens, etc.)